meteor/client/ui/SegmentList/SegmentListHeader.tsx

```diff
@@ -102,15 +103,23 @@ export function SegmentListHeader({
 					})}
 					tabIndex={0}
 				>
-					{playlist && parts && parts.length > 0 && (
-						<SegmentDuration
-							segmentId={segment._id}
-							parts={parts}
-							pieces={pieces}
-							label={<span className="segment-timeline__duration__label">{t('Duration')}</span>}
-							fixed={fixedSegmentDuration}
-						/>
-					)}
+					{playlist &&
+						parts &&
+						parts.length > 0 &&
+						(segment.isBreak ? (
+							<BreakWallTime
+								segment={segment}
+								label={<span className="segment-timeline__duration__label">{t('Wall Time')}</span>}
```

Comment:
I don't think "Wall time" is very descriptive. Should be something clearer, maybe "Expected start" or "Expected end"? I suppose if the anchor is `expectedStart`, the user may want to see the Segment Duration still, and this should replace the "On Air In/On Air At" countdown"?